osu.Game/Screens/Edit/IEditorChangeHandler.cs

```diff
@@ -10,6 +11,11 @@ namespace osu.Game.Screens.Edit
     /// </summary>
     public interface IEditorChangeHandler
     {
+        /// <summary>
+        /// Fired whenever a state change occurs.
+        /// </summary>
+        public event Action OnStateChange;
```

Comment:
```suggestion
        event Action OnStateChange;
```
Unnecessary, public by default.